ui/StatusQ/CMakeLists.txt

```diff
@@ -2,10 +2,14 @@ cmake_minimum_required(VERSION 3.19)
 
 project(StatusQ)
 
+option(STATUSQ_BUILD_SANDBOX "Enable to build StatusQ Sandbox application" ON)
+option(STATUSQ_BUILD_SANITY_CHECKER "Enable to build StatusQ Sanity Checker application" ON)
+option(STATUSQ_BUILD_TESTS "Enable to build StatusQ UI auto tests" ON)
+
 if (APPLE)
     set(MACOS_VERSION_MIN_FLAGS -mmacosx-version-min=10.14)
     set(CMAKE_OSX_ARCHITECTURES "x86_64")
```

Comment:
Why hardcode the arch here? :)

Reply:
(I know it's existing code build it should not be that way)

Reply:
Agree, time to drop this 🙂